lib/components/SPagination.vue

```diff
@@ -49,11 +51,11 @@ const hasNext = computed(() => {
 })
 
 function prev() {
-  hasPrev.value && emit('prev')
+  !props.disabled && hasPrev.value && emit('prev')
 }
 
 function next() {
-  hasNext.value && emit('next')
+  !props.disabled && hasNext.value && emit('next')
```

Comment:
this is already handled by SButton, can be removed from here?

https://github.com/globalbrain/sefirot/blob/ceb5fbfdfe53649ed659f59753937310a78ec90f/lib/components/SButton.vue#L90